lib/topic.ts

```diff
@@ -0,0 +1,39 @@
+import { expect, Page } from '@playwright/test';
+import { waitForKafkaReady } from './kafka';
+
+export const navigateToKafkaTopicsList = async function (page: Page, kafkaName: string) {
+  await expect(page.getByText(kafkaName)).toHaveCount(1);
+  await waitForKafkaReady(page, kafkaName);
+  await page.getByText(kafkaName).click();
+  await page.getByText('Topics').click();
+};
+
+export const createKafkaTopic = async function (page: Page, name: string, check = true) {
+  await page.getByText('Create topic').click();
+  await expect(page.getByText('Create topic')).toHaveCount(2);
+  // This is default Topic creation
+  await page.getByPlaceholder('Enter topic name').fill(name);
+  await page.getByText('Next').click();
+  await page.getByText('Next').click();
+  await page.getByText('Next').click();
+  await page.getByText('Finish').click();
+
+  if (check) {
+    // check that the instance has been created
+    const table = page.locator('[data-ouia-component-id=card-table]');
+    expect(table.getByText(name)).toBeTruthy();
+  }
+};
+
+export const deleteKafkaTopic = async function (page: Page, name: string) {
+  const instanceLinkSelector = page.getByText(name);
+  const row = page.locator('tr', { has: instanceLinkSelector });
+
+  await row.locator('[aria-label="Actions"]').click();
+  await page.getByText('Delete').click();
+  await page.getByLabel('Type DELETE to confirm:').click();
+  await page.getByLabel('Type DELETE to confirm:').fill('DELETE');
+  await page.getByTestId('modalDeleteTopic-buttonDelete').click();
+  await page.getByTestId('mk--instance__drawer').getByText('Kafka Instances').click();
+  await page.waitForURL('https://console.redhat.com/application-services/streams/kafkas');
```

Comment:
can we have here a check that is not dependent on the environment?
e.g. can we just check something in the page?

Reply:
I think that this check:
https://github.com/redhat-developer/consoledot-e2e/pull/23/files#diff-c8b5d72b55ebb6d6c1e1442604e109a1ca4d97d984c6e2c4b452c85271b98592R39
might be a good compromise.

Reply:
changed committed